Dockerfile

```diff
@@ -1,17 +1,21 @@
-FROM quay.io/fedora/fedora:38
+FROM golang:1.21 as builder
 
-RUN cd /usr/bin && \
-	curl -L https://github.com/kubevirt/kubevirt/releases/download/v1.0.0/virtctl-v1.0.0-linux-amd64 --output virtctl && \
-	chmod +x virtctl && \
-	curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
-	chmod +x kubectl
-RUN curl -LO https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz && \
-    tar -xzf openshift-client-linux.tar.gz -C /usr/local/bin/ && \
-    rm -f openshift-client-linux.tar.gz
+WORKDIR /app
+COPY go.mod go.sum ./
+COPY vendor/ ./vendor
+COPY main.go .
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -o kubevirt-disk-uploader .
 
-RUN dnf install -y buildah qemu-img && \
-	dnf clean all -y
+FROM quay.io/fedora/fedora:38
```

Comment:
We have 39 already. What about using a minimal fedora image?

Reply:
Okay let's use `fedora-minimal:39` and `microdnf` inside it.

Reply:
Can you install these packages in this Dockerfile for now?

Reply:
I'll change base image in a new PR https://github.com/codingben/kubevirt-disk-uploader/pull/7.